internal/ingress/controller/nginx.go

```diff
@@ -829,12 +828,7 @@ func configureCertificates(pcfg *ingress.Configuration, port int) error {
 }
 
 func post(url string, data interface{}) error {
-	buf, err := json.Marshal(data)
-	if err != nil {
-		return err
-	}
```

Comment:
Why are you deleting this? It's actually the payload being posted below at line 833.